src/include/daos/tse.h

```diff
@@ -34,9 +34,9 @@
  * tse_task is used to track single asynchronous operation.
  * 512 bytes all together.
  */
-#define TSE_TASK_SIZE		1024
+#define TSE_TASK_SIZE		1536
```

Comment:
This isn't used anywhere.  Is it just documenting?   Would it be better to define one in terms of the other?

#define TSE_TASK_SIZE 1536
#define TSE_PRIV_SIZE (TSE_TASK_SIZE - 8)

Or perhaps TSE_PUBLIC_SIZE 8 //or sizeof(public struct)

Reply:
OK. I will update the patch.
On the other hand, I have to merge this patch into the patch PR#811. We make this patch because we hit DAOS-2846 during PR#811 Jenkins test. But if without PR#811 patch, the test for this patch hit failure for DAOS-2884. So I have to merge the two patches together, then can pass the Jenkins test.